src/loose/state.js

```diff
@@ -158,4 +158,10 @@ export class LooseParser {
       plugin(this, pluginConfigs[name])
     }
   }
+
+  parse() {
+    let node = this.options.program || this.startNode()
+    this.next()
```

Comment:
I noticed in the default parser the call is `this.nextToken()` but I saw that `parse_dammit` uses `this.next`.
I'm not sure which is best.